lib/mappings/selectors.js

```diff
@@ -63,6 +71,12 @@ export function selectMultisigWalletAccounts(state, wallet) {
   return namespace[wallet];
 }
 
+export function selectTemporarySecrets(state, key) {
+  const namespace = selectWalletsNamespace(state, TEMPORARY_SECRETS_NAMESPACE);
+  if (key) return namespace[key];
```

Comment:
Thinking of refactoring the `if(key)...` because it will result in a polymorphic return value sometimes when `key` is undefined and `namespace[key]` is a `[]`